tests/erica_legacy/samples/grundsteuer_sample_input.json

```diff
@@ -38,7 +40,7 @@
           "wirtschaftliche_einheit_zaehler": "2.0000",
           "wirtschaftliche_einheit_nenner": "4"
         },
-        "groesse_qm": 100000000000000
+        "groesse_qm": 1000000000
```

Comment:
I have to admit, I don't get the regex, but the "Max Länge" field is pretty clear. Plus, I wouldn't have noticed if I didn't get an error before.

Fun Fact: The number I'm using here now: "1000000000" actually has 10 digits.....so it shouldn't work either...I'm just gonna change it to 9 as this is what we're gonna test for in the webapp anyways.

<img width="660" alt="image" src="https://user-images.githubusercontent.com/9320199/160071807-6cbe0d72-f81a-4f39-b853-6ad52d433062.png">

Reply:
The regex you posted is for E7403010, which is the (calculated) total area. The field above, groesse_qm, is E7411001 (Fläche des Flurstücks), which does have a max length of 15:

<img width="677" alt="image" src="https://user-images.githubusercontent.com/4248398/160092913-e339ed1b-ccc2-48ce-8927-f9dc8716c2bc.png">

(Your screenshot is already helpful though since it made me realize that my previous statement about E7403010 was wrong, it does have a max length of 9 indeed as indicated by the odd regex. This makes the overflow problem even worse though!)

Reply:
That's fine for now, but we need to make an explicit decision about this together with product. I'll talk to Katja.